cstrml/swork/src/mock.rs

```diff
@@ -393,23 +380,27 @@ pub fn add_success_sorders(who: &AccountId) {
     ].to_vec();
 
     for (idx, file) in files.iter().enumerate() {
-        insert_sorder(who, file, idx as u8, 350, OrderStatus::Success);
+        insert_sorder(who, file, idx as u8, 1000, OrderStatus::Success);
```

Comment:
Why change this?

Reply:
There is a new mechanism that "If the current block is lager than the sorder's expired_on, report_works won't update the sorder's status anymore.". We can double discuss about this behavior which make senses to me.

Reply:
There is a new mechanism that "If the current block is lager than the sorder's expired_on, report_works won't update the sorder's status anymore.". We can double discuss about this behavior which make senses to me.